systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2076 @@
+        items:
+          $ref: '#/definitions/Error'
+
+  Operation:
+    description: An operation provided by the API
+    type: object
+    properties:
+      available:
+        type: boolean
+        description: Whether the operation is available to the caller.
+        example: true
+      version:
+        type: integer
+        description: The version of the available operation.
+        example: 1
+    required: [available]
+
+  V1Operations:
+    description: V1 operations
+    type: object
+    properties:
+      operations:
+        description: >-
+          Available operations in the v1 version of the API:
+
+          - getJobs: The ability to retrieve the jobs
+          
+          - getSystems: The ability to retrieve the systems information
+
+          - getGroups: The ability to retrieve the groups
+    
+          - manageSystems: The ability to register/unregister systems
+          
+          - createOrCancelJobs: The ability to create or cancel jobs
+          
+          - activateSystems: The ability to activate systems
+          
+          - updateSystemMetadata: The ability to update systems metadata
+          
+          - manageFeeds: The ability to configure feeds and retrieve the available packages
+          
+          - createOrUpdateGroups: The ability to create, update or delete groups
+          
+        type: object
+        properties:
+          getJobs:
+            $ref: '#/definitions/Operation'
+          getSystems:
+            $ref: '#/definitions/Operation'
+          getGroups:
+            $ref: '#/definitions/Operation'
+          manageSystems:
+            $ref: '#/definitions/Operation'
+          createOrCancelJobs:
+            $ref: '#/definitions/Operation'
+          activateSystems:
+            $ref: '#/definitions/Operation'
+          updateSystemMetadata:
+            $ref: '#/definitions/Operation'
+          manageFeeds:
+            $ref: '#/definitions/Operation'
+          createOrUpdateGroups:
+            $ref: '#/definitions/Operation' 
+
+  QueryRequest:
+    description: Request for querying resources. More info here -> https://github.com/StefH/System.Linq.Dynamic.Core/wiki/Dynamic-Expressions and https://stefh.github.io/System.Linq.Dynamic.Core/html/144078bd-d1ad-bece-4daa-7fdd3a7ed7dc.htm
+    title: Query Request
+    properties:
+      skip:
+        description: How many resources to skip.
+        type: integer
+        default: 0
+        example: 100
+      take:
+        description: How many resources to take. Maximum value is 1000.
+        type: integer
+        default: -1
+        maximum: 1000
+        example: 100
+      filter:
+        type: string
+        example: connected.data.state==Connected
+        description: Specifies the filter criteria for resources.
+      projection:
+        type: string
+        example: 'new(id,grains,last_updated_timestamp)'
+        description: Specifies the projection for resources.
```

Comment:
What does "projection" mean here? Are we creating the resource request with this parameter?